src/sauce.config.js

```diff
@@ -47,6 +47,13 @@ const overrides = {
   testIgnore: process.env.TEST_IGNORE,
 };
 
+if (process.env.BROWSER_NAME !== 'chrome') {
+  overrides.use.browserName = process.env.BROWSER_NAME; // override browserName with suite browserName
+} else {
+  overrides.use.channel = 'chrome';
+  overrides.use.launchOptions.executablePath = process.env.BROWSER_PATH;
```

Comment:
It'd be good to explain what's going on here, especially the _why_. I don't think anyone will be able to tell in a few months.

To sum it up:
1. `chromium`, `firefox` and `webkit` come pre-packaged with playwright. So we can just pass those browser values to playwright and it knows what to do and where to pick them up.
2. In case of (google) chrome, it's somewhere on the VM. So we have to let playwright know where to look.

Reply:
sure. Let me add some comments